src/VisualStudio/Core/Def/ProjectSystem/VisualStudioProjectFactory.cs

```diff
@@ -77,6 +80,8 @@ public async Task<ProjectSystemProject> CreateAndAddToWorkspaceAsync(
 
         var vsixAnalyzerProvider = await _vsixAnalyzerProviderFactory.GetOrCreateProviderAsync(cancellationToken).ConfigureAwait(false);
 
+        _visualStudioWorkspaceImpl.LazyInsertedAnalyzerProvider = await _insertedAnalyzerProviderFactory.GetOrCreateProviderAsync(cancellationToken).ConfigureAwait(false);
```

Comment:
I wasn't sure where to put this "discovery of VSIX analyzers" (I'm not very familiar with the IDE part of the codebase) - I need that to be done before the `RedirectingAnalyzerAssemblyResolver` is called. I chose this place because it's where the similar `IVisualStudioDiagnosticAnalyzerProviderFactory` is called.